src/cli/grain.js

```diff
@@ -90,18 +55,16 @@ const grainCommand: Command = async (args, std) => {
   );
 
   distributions.map((d) => {
-    console.log(distributionMarkdownSummary(d, ledger, currencyDetails));
+    console.log(
```

Comment:
Let use the `std.out` param here instead of console.log

Reply:
accepted, changing in other files too